eng/test-rebuild.ps1

```diff
@@ -51,6 +51,7 @@ try {
 # Rebuilds with compilation errors
 # Rebuilds with missing references
 # Rebuilds with other issues
+  " --exclude net472\Microsoft.CodeAnalysis.EditorFeatures2.UnitTests.dll" +
```

Comment:
Was this change necessitated by other changes in the PR?

Reply:
No, I think there is a flaky issue there. I saw it pop up on a few other PRs. I'm looking at the diff locally right now to seew hat's going on.